src/manifolds/Stiefel.jl

```diff
@@ -194,7 +194,7 @@ function inverse_retract!(::Stiefel{n,k}, X, p, q, ::QRInverseRetraction) where
     for i in 1:k
         b = zeros(i)
         b[i] = 1
-        b[1:(end - 1)] = -transpose(R[1:(i - 1), 1:(i - 1)]) * A[i, 1:(i - 1)]
+        b[1:(end - 1)] = real(-transpose(R[1:(i - 1), 1:(i - 1)]) * A[i, 1:(i - 1)])
```

Comment:
@kellertuer This conversion of complex -> real was exact and thus non-throwing on Julia 1.0-1.4 but nightly has a small inexactness. Is it better to throw away the complex part or make `b` have the same eltype as `R`?

Reply:
I think I would prefer the `eltype`, since for a complex case, I am not 100% sure this will stay just real.